### PR TITLE
(LVM)Factory cleanups

### PR DESCRIPTION
**Pull request by vpodzime**
These are a few improvements and fixes for the generic ``DeviceFactory`` and ``LVMFactory`` classes. The commits are intentionally small and isolated so that it's easy to reject/skip or later revert any of them.

### CODE REVIEW

File: blivet/devicefactory.py [hunk 1]
```diff
@@ -1244,7 +1248,11 @@ def _get_device_size(self):
         return size
 
     def _set_device_size(self):
+        """ Set the size of the factory device. """
         size = self._get_device_size()
+
+        # self.raw_device = self.device.raw_device
```
Comment (dwlehman) [inline blivet/devicefactory.py]: maybe use `==` so it's obvious this is a statement rather than a commented-out assignment

Comment (vpodzime) [inline blivet/devicefactory.py]: Good point.

**Comment by vpodzime** [comment]
Jenkins, test this, please.

**Comment by vpodzime** [comment]
There are some pylint failures not related to this PR (but rather to some recent changes in https://github.com/rhinstaller/anaconda)

**Pull request merged**